### DISCUSSION TITLE
Confirming Unity support?

### DISCUSSION THREAD

**Discussion by KTRosenberg** [amerkoleci/JoltPhysicsSharp#4] [Announcements]
Since this is C#, is there anything that would keep me from integrating with Unity Engine projects? The advantage would be to support cross-engine physics with similar behaviors and APIs.

I realize that the native interface might work, but that wouldn’t be as simple for getting started.

**Comment by amerkoleci** [comment]
Hi, the library supports netstandard2.0 so it should work in unity as well, you can try and report any issues or submit PR in case of changes.

**Reply by KTRosenberg** [reply]
Thanks.
